CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
It might be good somewhere in here to say that any changes made to a repository will be licensed the same as the repository itself, and that by contributing, the contributor agrees to that